__tests__/index.spec.js

```diff
@@ -137,7 +145,7 @@ describe('one-app-dev-cdn', () => {
     jest
       .resetAllMocks()
       .resetModules();
-
+    removeModuleFromCache.mockImplementation(() => ({}));
```

Comment:
Is this needed? this should already by the mock implementation of this function at this point?

Reply:
Yes, cache is initialized once and is used on each network request. Existing test case uses same key, so we don't return empty cache, existing test case will fail as it returns from cache.